### PR TITLE
Bugfix page mask detail button when empty

### PR DESCRIPTION
**Pull request by Hamsolovski**

### CODE REVIEW

**Review by WildAntho — APPROVED** [review]

**Review by Alexandre78R — APPROVED** [review]
Good

File: client/src/pages/TemplateDetails/templateDetails.css [hunk 1]
```diff
@@ -83,3 +83,12 @@
   box-shadow: none;
   align-self: flex-end;
 }
+
+.templatedetail-logo-type {
```
Comment (Alexandre78R) [inline client/src/pages/TemplateDetails/templateDetails.css:87]: template-detail ?

File: client/src/pages/TemplateDetails/templateDetails.css [hunk 2]
```diff
@@ -83,3 +83,12 @@
...
+  font-size: 1.4em;
+  color: var(--template-dark);
+}
+
+.templatedetail-title {
```
Comment (Alexandre78R) [inline client/src/pages/TemplateDetails/templateDetails.css:92]: template-detail ?

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

**Review by Alexandre78R — COMMENTED** [review]
.

**Pull request merged**